cg_hermes/deliverables.py

```diff
@@ -68,6 +70,11 @@ def set_pipeline_specific_variables(self):
             )
             self.files = self.get_microsalt_files()
             self.configs = Deliverables.build_internal_tag_map(MICROSALT_COMMON_TAGS)
+        elif self.pipeline == Pipeline.microsalt:
```

Comment:
```suggestion
        elif self.pipeline == Pipeline.mutant:
```
It fails in the way it does since it uses MIP as fallback. It might have been easier to track the error if it dies when pipeline option was missing. Perhaps you could change that?